### ISSUE TITLE
Build fails in Cargo workspace

### ISSUE THREAD

**Issue by nklayman** [tauri-apps/tauri#1007]
**Describe the bug**
If the Tauri project is inside of a Cargo workspace, the target dir will be different then usual. The bundle will fail when it tries to create installers as they look in `src-tauri/target` for the executable when it is really up 2 levels from that due to the workspace dist.

**To Reproduce**
1. Create a tauri project inside of a Cargo workspace
2. Try to build the project

**Expected behavior**
It should either output the executable to `src-tauri/target` and ignore the workspace dist dir or it should look in the proper dir for the executable when bundling the installers.

**Screenshots**
![image](https://user-images.githubusercontent.com/12448243/91919924-5dadea00-ec7c-11ea-8db9-1fdba9df636e.png)

**Platform and Versions (please complete the following information):**
Any

**Additional context**
https://github.com/tauri-apps/vue-cli-plugin-tauri/pull/20

**Comment by victorhqc** [comment]
Having the same issue, tried to workaround it by adding a `.cargo/config` file in the same path that the tauri application

```toml
[build]
target-dir = "src-tauri/target"
```

But couldn't make it work 😢 had to move all my code to its own repository for now.

**Update**

Made it work by using full path in `.cargo/config`

1.  Do `pwd` to get full path

1.  Modify `.cargo/config` and replace `<pwd>` with full path for repository targeting the correct `src-tauri` location.

    ```toml
    [build]
    target-dir = "<pwd>/src-tauri/target"
    ```

**Comment by x7c1** [comment]
I've found similar workaround which doesn't need embedding the full path.

Instead of modifying `.cargo/config`, you can add an [environment variable](https://doc.rust-lang.org/cargo/reference/config.html#buildtarget-dir) in `package.json` :

```json
  "scripts": {
    "tauri": "CARGO_BUILD_TARGET_DIR=$(pwd)/src-tauri/target tauri",
```

`tauri build` will be able to find `src-tauri/target` as usual :+1: 

```
npm run tauri build
```

**Comment by lucasfernog** [comment]
The tauri bundler has some logic to detect workspaces, I guess it's not working properly. I'll investigate.

**Comment by lucasfernog** [comment]
Closed by #1235
